Automattic/wp-calypso
# Signup: remove Pro and Starter flows

Related to #

## Proposed Changes

* This PR removes the `/start/pro` and `/start/starter` flows. Users visiting either of those flows will be automatically redirected to `/start`.

## Why are these changes being made?


This is a part of pau2Xa-605-p2. The Pro and Starter(Legacy) plans have been discontinued, and these flows do not result in plan purchases. (34d61-pb/#plain)

## Testing Instructions



* Go to `/start/pro` and `/start/starter` and confirm that you are redirected to `/start`.

## Pre-merge Checklist
